CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 21.2.2
+
+Add `--only` and `--ignore` options to `openfisca-run-test` to filter out tested output variables if needed.
```

Comment:
`--only` seems a little too general. `-n` also _only tests with the given name_filter in their name_. What about `--only-variables` and `--ignore-variables` ?